index.Rmd

```diff
@@ -153,24 +153,15 @@ The other functions are optional, with default values used if they are not speci
 1. Run `ggplot2::ggplot(data = mpg)` what do you see?
 2. What does the `drv` variable describe? Read the help for mpg data set to find out by running `?mpg`
 3. Make a scatter plot of `hwy` vs. `cty`
-4. Make a scatter plot of `class` vs. `drv`. Why isn't this plot useful?
-
-### Extension
-
-To fix the problem with exercise 4, make a scatter plot of `class` vs `cty` and colour the points by 
-`drv` (more on this in the next section)
+4. Make a scatter plot of urban fuel consumption vs engine size (displacement). 
 
 ### Note
 
-The first time you plot something in each R session on the Analytical Platform you may get the 
-following warning. You can ignore it.
+You will need to run the following first to import the necessary functions and data:
 
-```
-Warning message:
-In grSoftVersion() :
-  unable to load shared object '/usr/local/lib/R/modules//R_X11.so':
-  libXt.so.6: cannot open shared object file: No such file or directory
-```
+`import::from("ggplot2", "aes", "vars", "mpg")`
+
+`import::from("magrittr", "%>%")`
```

Comment:
For interactive work it is simpler to import the packages via `library()`. I might change this to the below (although it is a bit long).

```
To import the necessary functions and data you will need to run:

`import::from("ggplot2", "aes", "vars", "mpg")`
`import::from("magrittr", "%>%")`

or

`library(ggplot2)`
`library(magrittr)`
```